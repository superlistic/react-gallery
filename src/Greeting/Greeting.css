.greeting {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  width: 100vw;
  height: 100vh;
  background-size: cover;
  background-image: url('https://unsplash.com/photos/mMSNRZvaiCU/download?force=true&w=1920');
}
.greeting__text {
  color: var(--clr-bright);
}
