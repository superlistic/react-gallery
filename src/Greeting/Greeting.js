import React from 'react';
import '../main.css';
import './Greeting.css';

const Greeting = () => {
  return (
    <div className="greeting">
      <h5 className="greeting__text">Welcome to our React Gallery!</h5>
      {/* <br /> */}
      <p className="greeting__text">Search below for awesome pictures</p>
    </div>
  );
};

export default Greeting;
