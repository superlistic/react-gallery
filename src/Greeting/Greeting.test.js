import Greeting from './Greeting';
import React from 'react';
import renderer from 'react-test-renderer';

test('renders elements', () => {
  const tree = renderer.create(<Greeting />).toJSON();
  expect(tree).toMatchSnapshot();
});
