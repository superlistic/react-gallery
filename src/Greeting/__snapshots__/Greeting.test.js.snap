// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders elements 1`] = `
<div
  className="greeting"
>
  <h1
    className="greeting__heading"
  >
    re:splash
  </h1>
  <h2
    className="greeting__subheading"
  >
    "By the power of unsplash you can.."
  </h2>
</div>
`;
